src/variables.ts

```diff
@@ -41,26 +41,30 @@ export const generateVariables = async (
     org,
     repo,
     buildOrg,
-    buildRepo,
+    buildRepo
   } = inputs
 
   const outputs: {[key: string]: string} = {}
 
   const artifactoryDockerRegistryHostname = `${artifactoryOrg}-${artifactoryDockerRepository}.jfrog.io`
 
-  outputs['org'] = org,
-  outputs['repo'] = repo,
-  outputs['build_number'] = `${sha}:${runId}`
+  ;(outputs['org'] = org),
```

Comment:
Is this correct? line starting with `;` ?

Reply:
No, that was moved by eslint when cleaning code. Let me run the clean task again.